frontends/infinite-corridor/package.json

```diff
@@ -20,13 +20,14 @@
     "@swc/core": "^1.3.32",
     "@tanstack/react-query": "^4.29.5",
     "@tanstack/react-query-devtools": "^4.29.6",
-    "@testing-library/react-hooks": "^8.0.1",
     "@types/lodash": "^4.14.182",
-    "@types/react": "^18.0.13",
-    "@types/react-dom": "^18.0.5",
+    "@types/react": "^18.2.6",
+    "@types/react-dom": "^18.2.4",
     "@types/react-router": "^5.1.18",
     "@types/react-router-dom": "^5.3.3",
+    "autoprefixer": "^10.4.14",
     "axios": "^0.27.2",
+    "bootstrap": "^5.2.3",
```

Comment:
We were using this before, it just wasn't explicitly listed as a dependency.

We aren't using it very much: only for utility classes, and only in a few places. We may want to remove it.